server/repository/odp/listOriginalDataPoints.ts

```diff
@@ -0,0 +1,17 @@
+import * as db from '@server/db/db'
+import * as Promise from 'bluebird'
```

Comment:
not sure if we should still use Bluebird. what do you think?

Reply:
We can remove, but in this file it was not straightforward to replace iirc
(Would prefer removing)

Reply:
please, remove it. future refactor will remove it as well